portability-worker/src/main/java/org/dataportabilityproject/worker/WorkerModule.java

```diff
@@ -61,6 +58,9 @@
 
   @Override
   protected void configure() {
+    // binds flags from ExtensionContext to @Named annotations
+    super.configure();
```

Comment:
What if we did FlagBindingModule.parse(context) instead of in the construtor

Reply:
Done (I think - let me know if this isn't what you meant)